### DISCUSSION TITLE
Completing a checkout without payment

### DISCUSSION THREAD

**Discussion by joeydeckers** [medusajs/medusa#4993] [Q&A]
Hi all,

We're looking into using Medusajs for a client project. That want to have an internal webshop for their large corporate company. But one of the requirements is that customers don't have to pay in order to place an order. Because customers are employees of that company.

Hopefully this is possible, thanks in advance!.

**Comment by Labham-Jain** [comment]
I believe you can use medusa-payment-manual for that case
